Fix for the failure of trigger e2e test

daisy-ycguo opened the pull request:
## Proposed Changes

* Remove labels and triggers before ending the e2e test

navidshaikh commented on test/e2e/trigger_test.go:
```diff
@@ -40,13 +40,17 @@ func TestBrokerTrigger(t *testing.T) {
 
 	err = test.lableNamespaceForDefaultBroker(t)
 	assert.NilError(t, err)
+	defer unlableNamespaceForDefaultBroker(t)
```
```suggestion
	defer test.unlableNamespaceForDefaultBroker(t)
```

daisy-ycguo commented:
/test pull-knative-client-integration-tests

navidshaikh commented:
/test pull-knative-client-integration-tests

daisy-ycguo commented:
/test pull-knative-client-integration-tests

daisy-ycguo commented:
@navidshaikh will you check if you are OK with the changes to trigger e2e test ? I removed label and deleted triggers before shutting down the test environment to avoid flaky errors.

matzew commented on test/e2e/trigger_test.go:
```diff
@@ -58,11 +62,19 @@ func TestBrokerTrigger(t *testing.T) {
 	test.triggerCreateMissingSink(t, r, "errtrigger", "notfound")
 }
 
+func (test *e2eTest) unlableNamespaceForDefaultBroker(t *testing.T) {
```
/lgtm
/approve

navidshaikh commented:
@daisy-ycguo : Yes, lets merge this and unblock other PRs.

FYI @dsimansk logged https://github.com/knative/eventing/issues/2668 for this issue.
